install_config/storage_examples/shared_storage.adoc

```diff
@@ -13,4 +13,473 @@ toc::[]
+
+
+----
+<1> The project (namespace) name.
+<2> The IP address of the OpenShift node running the pod.
+<3> The PVC name used by the pod.
+<4> The list of events resulting in the pod being launched and the nfs volume being
+mounted. The container will not start correctly if the volume cannot mount.
+====
+
+There is more internal information, including the SCC used to authorize the pod, the pod's
+user and group ids, the selinux label, etc. shown in the
+`oc get pod <name> -o yaml` command:
+
+====
+----
+[root@ose70 nfs]# oc get pod nginx-nfs-pod -o yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    openshift.io/scc: restricted <1>
+  creationTimestamp: 2016-03-21T13:59:47Z
+  labels:
+    name: nginx-nfs-pod
+  name: nginx-nfs-pod
+  namespace: default <2>
+  resourceVersion: "2814411"
+  selfLink: /api/v1/namespaces/default/pods/nginx-nfs-pod
+  uid: 2c22d2ea-ef6d-11e5-adc7-000c2900f1e3
+spec:
+  containers:
+  - image: fedora/nginx
+    imagePullPolicy: IfNotPresent
+    name: nginx-nfs-pod
+    ports:
+    - containerPort: 80
+      name: web
+      protocol: TCP
+    resources: {}
+    securityContext:
+      privileged: false
+    terminationMessagePath: /dev/termination-log
+    volumeMounts:
+    - mountPath: /usr/share/nginx/html
+      name: nfsvol
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      name: default-token-a06zb
+      readOnly: true
+  dnsPolicy: ClusterFirst
+  host: ose70.rh7
+  imagePullSecrets:
+  - name: default-dockercfg-xvdew
+  nodeName: ose70.rh7
+  restartPolicy: Always
+  securityContext:
+    supplementalGroups:
+    - 100003i <3>
+  serviceAccount: default
+  serviceAccountName: default
+  terminationGracePeriodSeconds: 30
+  volumes:
+  - name: nfsvol
+    persistentVolumeClaim:
+      claimName: nfs-pvc <4>
+  - name: default-token-a06zb
+    secret:
+      secretName: default-token-a06zb
+status:
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: 2016-03-21T13:59:49Z
+    status: "True"
+    type: Ready
+  containerStatuses:
+  - containerID: docker://a3292104d6c28d9cf49f440b2967a0fc5583540fc3b062db598557b93893bc6f
+    image: fedora/nginx
+    imageID: docker://403d268c640894cbd76d84a1de3995d2549a93af51c8e16e89842e4c3ed6a00a
+    lastState: {}
+    name: nginx-nfs-pod
+    ready: true
+    restartCount: 0
+    state:
+      running:
+        startedAt: 2016-03-21T13:59:49Z
+  hostIP: 192.168.234.148
+  phase: Running
+  podIP: 10.1.0.4
+  startTime: 2016-03-21T13:59:47Z
+
+----
+<1> The SCC used by the pod.
+<2> The project (namespace) name.
+<3> The supplemental group ID for the pod (all containers).
+<4> The PVC name used by the pod.
+====
+
+=== 5. Create another Pod that uses the same claim
+A pod definition file or a template file can be used to define a pod. Below is a pod spec that
+creates a single container and mounts the NFS volume for read-write access:
```

Comment:
This paragraph is copy/paste of the pod paragraph above and seems "robotic" to me. How about:
_Now create a second pod, this one will run the nginx container. The pod definition is similar to the first pod and is shown below:_